specification/containerservices/resource-manager/Microsoft.ContainerService/stable/2018-03-31/managedClusters.json

```diff
@@ -1311,6 +1401,35 @@
         "modelAsString": true
       },
       "description": "OsType to be used to specify os type. Choose from Linux and Windows. Default to Linux."
+    },
+    "CredentialResults":{
+      "properties": {
+        "values": {
+          "type" : "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/CredentialResult"
+          }
+        },
+        "description": "The list of credential result."
```

Comment:
This line doesn't pass when I run `autorest`, as if it's not expecting `"description"` there:

```console
$ autorest $HOME/Projects/azure-rest-api-specs/specification/containerservices/resource-manager/readme.md --python --azure-validator --python-sdks-folder=$HOME/Projects/azure-sdk-for-python --python-mode=update
AutoRest code generation utility [version: 2.0.4282; node: v8.9.3]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest

There is a new version of AutoRest available (2.0.4283).
 > You can install the newer version with with npm install -g autorest@latest

   Loading AutoRest core      '/Users/matt/.autorest/@microsoft.azure_autorest-core@2.0.4282/node_modules/@microsoft.azure/autorest-core/dist' (2.0.4282)
   Loading AutoRest extension '@microsoft.azure/autorest.python' (~2.1.26->2.1.40)
   Loading AutoRest extension '@microsoft.azure/classic-openapi-validator' (~1.0.9->1.0.13)
   Loading AutoRest extension '@microsoft.azure/openapi-validator' (~1.0.2->1.0.4)
   Loading AutoRest extension '@microsoft.azure/autorest.modeler' (2.3.44->2.3.44)
ERROR: Schema violation: Expected type object but found type string
    - file:///Users/matt/Projects/azure-rest-api-specs/specification/containerservices/resource-manager/Microsoft.ContainerService/stable/2018-03-31/managedClusters.json:1414:8 ($.definitions.CredentialResults.properties.description)
FATAL: swagger-document/individual/schema-validator - FAILED
FATAL: Error: [OperationAbortedException] Error occurred. Exiting.
Process() cancelled due to exception : [OperationAbortedException] Error occurred. Exiting.
```